### ISSUE TITLE
Helm chart defines resources twice

### ISSUE THREAD

**Issue by estahn** [solo-io/gloo#3609]
**Describe the bug**

Using ArgoCD with the opensource helm chart we get the error message that manifests for resources are defined twice.

![image](https://user-images.githubusercontent.com/362174/92441368-6f017400-f1f1-11ea-9597-ce92eff7a9be.png)

**To Reproduce**
1. Run `helm template . | yq r -d'*' - '(.kind == ServiceAccount)'`
2. See 2 `certgen` service accounts

**Expected behavior**

Resource manifests should be defined only once.

**Additional context**
- Gloo version: >1.4.4
- Kubernetes version: 1.6
- Helm version: 3

**Comment by kdorosh** [comment]
closing as duplicate of https://github.com/solo-io/gloo/issues/3498
